### ISSUE TITLE
Sources/Javadoc missing

### ISSUE THREAD

**Issue by Takhion** [dlew/joda-time-android#25]
Hi, I noticed that sources and javadoc are missing, it is because joda-time is not exporting them to Maven central?

**Comment by dlew** [comment]
Yeah, I've noticed that, too. I'm not really an expert on gradle-to-maven so I'm not sure why it's not uploading (the javadoc gets created but not sent over).

**Comment by austynmahoney** [comment]
I think the issue might be that Android Studio is looking for the wrong file. This library uses the `no-tzdb` jar, so Android studio looks for javadoc/sources for `joda-time-2.7-no-tzdb.jar`, which don't exist on Maven Central.

Error message in AS:
`Sources for: joda-time-2.7-no-tzdb.jar not found`

**Comment by dlew** [comment]
I'm hoping f2602e0bedd7f7fd79f43dc79badd986245dabd7 will fix the sources problem (though I won't upload a new version to maven central until there's either a new version of joda-time or the tzdata).

As for the joda-time dependency - the normal sources.jar on maven central *should* work, since it's all the same code. Does anyone know if there's anything else I can do to point it in that direction?

**Comment by christianguevara** [comment]
I dont know if you updated a new version in maven central, but still no javadoc found :(

**Comment by dlew** [comment]
@christianguevara Javadoc for joda-time, or javadoc for the joda-time-android-specific classes (like `DateUtils`)?

I won't be able to include javadoc in joda-time until joda-time itself updates, which it hasn't for a while.

**Comment by user-don** [comment]
dlew, first off this is a fantastic library and I'm a total fan. Any update on whether it's possible to get the joda-time javadocs available for use when developing?

**Comment by PaulWoitaschek** [comment]
Any news on this? Its quite annoying to go to view the javadoc in a browser when finding out what the meaning of a signature is.

**Comment by xgouchet** [comment]
I'm also still waiting for this to be available

**Comment by dlew** [comment]
Perhaps https://github.com/JodaOrg/joda-time/pull/392 will fix it?

**Comment by PaulWoitaschek** [comment]
@dlew Can you leave this open until its fixed?

**Comment by dlew** [comment]
Fine with me. I think it's fixed now, though it's dependent on joda-time making another release, so we can leave it open until then.

**Comment by PaulWoitaschek** [comment]
Still present for me in 2.9.5

**Comment by benj56** [comment]
And 2.9.7.. it's really inconvenient, to the point where I use the regular jodatime when not doing a release build.

**Comment by PetarMarijanovic** [comment]
I really like this library, but it's hard to use it without the sources :S Are there any updates on this issue?

**Comment by xbao** [comment]
As a workaround you can download the javadoc jar from http://central.maven.org/maven2/joda-time/joda-time/2.9.9/joda-time-2.9.9-no-tzdb-javadoc.jar, right click on the `joda-time-2.9.9-no-tzdb-javadoc.jar` library under `External Libraries` in Android Studio, click `Library Properties` and add the javadoc jar

**Comment by TheFreezingChicken** [comment]
I've encountered this issue as well, and I started to dig into it a little.
I need to clarify that I'm a complete newbie to Maven and so what I'm saying might be stupid as it is just a result of manual searching and pattern-recognition without any knowledge.

It seems that in the pom file of the original joda-time repo the sources and docs are attached with 2 separate classifiers, namely `no-tzdb-javadoc` and `no-tzdb-sources`, but in the joda-time-android pom I only see the classifier for the main jar (which is `no-tzdb`).

To further support my newbie theory, if you go in the `.idea\libraries\Gradle__joda_time_joda_time_2_10_6_no_tzdb_jar.xml` file of a project that uses joda-time-android the tags `JAVADOC` and `SOURCES` are both empty, and if you go in the directory `.gradle/caches/modules-2/files-2.1/joda-time/joda-time/2.10.6/` of your user profile folder you will find the jar for the no-tzdb version alongside with the docs and sources of the normal version (the one without the appended `no-tzdb` at the end) instead of the correct one.

So I have no idea if the problem is on joda-time side or joda-time-android (or both) but I have a feeling it has to do with how those classifiers or dependencies are declared in the poms.
